src/ui/locales/id.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Alihkan atribusi",
+    "AttributionControl.MapFeedback": "Umpan balik peta",
+    "FullscreenControl.Enter": "Memasuki layar penuh",
```

Comment:
Minor, but for consistency 
```suggestion
    "FullscreenControl.Enter": "Masuk ke layar penuh",
```